src/xdocs/config_javadoc.xml

```diff
@@ -1631,6 +1631,43 @@ public boolean isSomething()
     &lt;property name=&quot;period&quot; value=&quot;STRING_LITERAL&quot;/&gt;
```

Comment:
@romani @rnveach 
This example is invalid.
See http://rveach.no-ip.org/checkstyle/checkstyle.php?action=view&config=abeb8b079dd5d1ed2d2c10f81fc65d12&code=0db8c02c469b9086140536de29d1f527&checkstyle=checkstyle-7.6-all.jar&printTree=

Should be
```xml
<module name="SummaryJavadocCheck">
    <property name="period" value="."/>
</module>
```

Looks like a copy-paste error.

@sagar-shah94 
Please, fix this example.